Explain why ES plugins removal does not deinstall them (#1628)
platformsh/platformsh-docs: src/configuration/services/elasticsearch.md

| mapper-size           | Size mapper plugin, enables the `_size` meta field                                        | *   | *   | *   | *   |
| repository-s3         | Support for using S3 as a repository for Snapshot/Restore                                 |     | *   | *   | *   |

### Plugins removal

> Removing plugins previously added in your `services.yaml` file will not automatically deinstall them from your Elasticsearch instances.  This is deliberate, as removing a plugin may result in data loss or corruption of existing data that relied on that plugin.  Removing a plugin will usually require reindexing.

Reviewer: ```suggestion
Removing plugins previously added in your `services.yaml` file will not automatically uninstall them from your Elasticsearch instances.  This is deliberate, as removing a plugin may result in data loss or corruption of existing data that relied on that plugin.  Removing a plugin will usually require reindexing.
```

| mapper-size           | Size mapper plugin, enables the `_size` meta field                                        | *   | *   | *   | *   |
| repository-s3         | Support for using S3 as a repository for Snapshot/Restore                                 |     | *   | *   | *   |

### Plugins removal

> Removing plugins previously added in your `services.yaml` file will not automatically deinstall them from your Elasticsearch instances, this may sound counter intuitive, but it's on purpose and the rationale with it is that deinstalling a plugin usually require carefull toughts about potential data-loss/corruption and almost certainly the need for re-indexing.

Reviewer 2: ```suggestion
Removing plugins previously added in your `services.yaml` file will not automatically deinstall them from your Elasticsearch instances.  This is deliberate, as removing a plugin may result in data loss or corruption of existing data that relied on that plugin.  Removing a plugin will usually require reindexing.
```

> If you have to remove a previously installed plugin for good, the recommended way is:
> 1. Rename the service
> 2. Reindex your data

Reviewer 2: ```suggestion
If you wish to permanently remove a previously-enabled plugin, you will need to follow the "Upgrading" procedure below to create a new instance of Elasticsearch and migrate to it.  In most cases that is not necessary, however, as an unused plugin has no appreciable impact on the server.
```
